src/classes/Update.php

```diff
@@ -36,7 +36,7 @@
 class Update
 {
     /** @var int REQUIRED_SCHEMA the current version of the database structure */
-    private const REQUIRED_SCHEMA = 57;
+    private const REQUIRED_SCHEMA = 58;
```

Comment:
use schema57 and merge the code in schema57.sql

Reply:
Changed in 4c4dbce